I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 246, in script call
    call expression tutorial.label from _call_expression
  File "game/script.rpy", line 246, in script call
    call expression tutorial.label from _call_expression
  File "game/eat.rpy", line 13, in script
    if Player.stats.currentHealth+3<=Player.stats.maxhealth:
  File "game/eat.rpy", line 13, in <module>
    if Player.stats.currentHealth+3<=Player.stats.maxhealth:
AttributeError: 'Stats' object has no attribute 'maxhealth'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 246, in script call
    call expression tutorial.label from _call_expression
  File "game/script.rpy", line 246, in script call
    call expression tutorial.label from _call_expression
  File "game/eat.rpy", line 13, in script
    if Player.stats.currentHealth+3<=Player.stats.maxhealth:
  File "/Applications/renpy-7.3.5-sdk/renpy/ast.py", line 1832, in execute
    if renpy.python.py_eval(condition):
  File "/Applications/renpy-7.3.5-sdk/renpy/python.py", line 2059, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "/Applications/renpy-7.3.5-sdk/renpy/python.py", line 2052, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/eat.rpy", line 13, in <module>
    if Player.stats.currentHealth+3<=Player.stats.maxhealth:
AttributeError: 'Stats' object has no attribute 'maxhealth'

Darwin-19.6.0-x86_64-i386-64bit
Ren'Py 7.3.5.606
Gates of Galloo 1.0
Mon Nov 30 14:23:16 2020
